spec/models/conference_spec.rb

```diff
@@ -778,6 +778,35 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+
+    before(:each) do
+      @conference = create(:conference)
```

Comment:
there is already a conference variable, that you can use: https://github.com/siddhantbajaj/osem/blob/b8af1854a4a255d62a4507e7607f01ae89b79f02/spec/models/conference_spec.rb#L7

Although I have no idea why it is called `subjetc`. Maybe we should rename it (if you want to do so in this PR please add it in a different commit) :thinking: